EscapedBranchName is not supported by GitVersion.CommandLine

### What You Are Seeing?

Upstream GitVersion has a [`EscapedBranchName` variable](https://gitversion.net/api/GitVersion.OutputVariables/VersionVariables/B3AFABF7) which is not available though the cake tool ([example output](https://gitversion.readthedocs.io/en/latest/input/docs/more-info/variables/)).

### What is Expected?

```csharp
    version = GitVersion(new GitVersionSettings { NoFetch = true });
    Information($"version.BranchName = {version.BranchName}");
    Information($"version.EscapedBranchName = {version.EscapedBranchName}");
```

should not fail.

### What version of Cake are you using?

Cake 0.38.5
GitVersion.CommandLine 5.6.6

### Are you running on a 32 or 64 bit system?

64bit Windows


### Output Log

```shell
powershell .\build.ps1 -Target ver -Verbosity Diagnostic

Preparing to run build script...
Running build script...
Module directory does not exist.
NuGet Config not specified. Will use NuGet default mechanism for resolving it.
Analyzing build script...
Analyzing C:/git/.../build.cake...
Processing build script...
Installing tools...
The 'tool' directive is attempting to install the 'GitVersion.CommandLine' package
without specifying a package version number.
More information on this can be found at https://cakebuild.net/docs/tutorials/pinning-cake-version
It's not recommended, but you can explicitly override this warning
by configuring the Skip Package Version Check setting to true
(i.e. command line parameter "--settings_skippackageversioncheck=true",
environment variable "CAKE_SETTINGS_SKIPPACKAGEVERSIONCHECK=true",
read more about configuration at https://cakebuild.net/docs/fundamentals/configuration)
  CACHE https://api.nuget.org/v3/registration5-gz-semver2/gitversion.commandline/index.json
  CACHE https://api.nuget.org/v3/registration5-gz-semver2/gitversion.commandline/page/1.1.0/4.0.1-beta1-32.json
  CACHE https://api.nuget.org/v3/registration5-gz-semver2/gitversion.commandline/page/4.0.1-beta1-33/5.0.0-beta2-23.json
  CACHE https://api.nuget.org/v3/registration5-gz-semver2/gitversion.commandline/page/5.0.0-beta2-25/5.0.0-beta4-11.json
  CACHE https://api.nuget.org/v3/registration5-gz-semver2/gitversion.commandline/page/5.0.0-beta4-12/5.0.1-beta1.11.json
  CACHE https://api.nuget.org/v3/registration5-gz-semver2/gitversion.commandline/page/5.0.1-beta1.13/5.1.1-beta1.1.json
  CACHE https://api.nuget.org/v3/registration5-gz-semver2/gitversion.commandline/page/5.1.1/5.1.4-beta1.7.json
  CACHE https://api.nuget.org/v3/registration5-gz-semver2/gitversion.commandline/page/5.1.4-beta1.11/5.1.4-beta1.177.json
  CACHE https://api.nuget.org/v3/registration5-gz-semver2/gitversion.commandline/page/5.1.4-beta1.179/5.6.6.json
Installing addins...
The addin Cake.Yarn will reference Cake.Yarn.dll.
The addin Cake.Docker will reference Cake.Docker.dll.
Verifying assembly 'Cake.Yarn, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
Verifying assembly 'Cake.Docker, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'.
Adding assembly reference to mscorlib.dll...
Adding assembly reference to System.Core.dll...
Adding assembly reference to Microsoft.CSharp.dll...
Adding assembly reference to Cake.Core.dll...
Adding assembly reference to Cake.Common.dll...
Adding assembly reference to Cake.exe...
Adding assembly reference to System.dll...
Adding assembly reference to System.Xml.dll...
Adding assembly reference to System.Xml.Linq.dll...
Adding assembly reference to System.Data.dll...
Adding assembly reference to System.Runtime.dll...
Adding assembly reference to System.Collections.dll...
Adding assembly reference to System.Net.Http.dll...
Adding assembly reference to netstandard.dll...
Adding assembly reference to Cake.Yarn.dll...
Adding assembly reference to Cake.Docker.dll...
Importing namespace Cake.Common...
Importing namespace Cake.Common.Build...
Importing namespace Cake.Common.Build.AppVeyor...
Importing namespace Cake.Common.Build.AppVeyor.Data...
Importing namespace Cake.Common.Build.AzurePipelines...
Importing namespace Cake.Common.Build.AzurePipelines.Data...
Importing namespace Cake.Common.Build.Bamboo...
Importing namespace Cake.Common.Build.Bamboo.Data...
Importing namespace Cake.Common.Build.BitbucketPipelines...
Importing namespace Cake.Common.Build.BitbucketPipelines.Data...
Importing namespace Cake.Common.Build.Bitrise...
Importing namespace Cake.Common.Build.Bitrise.Data...
Importing namespace Cake.Common.Build.ContinuaCI...
Importing namespace Cake.Common.Build.ContinuaCI.Data...
Importing namespace Cake.Common.Build.GitHubActions...
Importing namespace Cake.Common.Build.GitHubActions.Data...
Importing namespace Cake.Common.Build.GitLabCI...
Importing namespace Cake.Common.Build.GitLabCI.Data...
Importing namespace Cake.Common.Build.GoCD...
Importing namespace Cake.Common.Build.GoCD.Data...
Importing namespace Cake.Common.Build.Jenkins...
Importing namespace Cake.Common.Build.Jenkins.Data...
Importing namespace Cake.Common.Build.MyGet...
Importing namespace Cake.Common.Build.TeamCity...
Importing namespace Cake.Common.Build.TFBuild...
Importing namespace Cake.Common.Build.TFBuild.Data...
Importing namespace Cake.Common.Build.TravisCI...
Importing namespace Cake.Common.Build.TravisCI.Data...
Importing namespace Cake.Common.Diagnostics...
Importing namespace Cake.Common.IO...
Importing namespace Cake.Common.IO.Paths...
Importing namespace Cake.Common.Net...
Importing namespace Cake.Common.Security...
Importing namespace Cake.Common.Solution...
Importing namespace Cake.Common.Solution.Project...
Importing namespace Cake.Common.Solution.Project.Properties...
Importing namespace Cake.Common.Solution.Project.XmlDoc...
Importing namespace Cake.Common.Text...
Importing namespace Cake.Common.Tools...
Importing namespace Cake.Common.Tools.Cake...
Importing namespace Cake.Common.Tools.Chocolatey...
Importing namespace Cake.Common.Tools.Chocolatey.ApiKey...
Importing namespace Cake.Common.Tools.Chocolatey.Config...
Importing namespace Cake.Common.Tools.Chocolatey.Download...
Importing namespace Cake.Common.Tools.Chocolatey.Features...
Importing namespace Cake.Common.Tools.Chocolatey.Install...
Importing namespace Cake.Common.Tools.Chocolatey.New...
Importing namespace Cake.Common.Tools.Chocolatey.Pack...
Importing namespace Cake.Common.Tools.Chocolatey.Pin...
Importing namespace Cake.Common.Tools.Chocolatey.Push...
Importing namespace Cake.Common.Tools.Chocolatey.Sources...
Importing namespace Cake.Common.Tools.Chocolatey.Uninstall...
Importing namespace Cake.Common.Tools.Chocolatey.Upgrade...
Importing namespace Cake.Common.Tools.DotCover...
Importing namespace Cake.Common.Tools.DotCover.Analyse...
Importing namespace Cake.Common.Tools.DotCover.Cover...
Importing namespace Cake.Common.Tools.DotCover.Merge...
Importing namespace Cake.Common.Tools.DotCover.Report...
Importing namespace Cake.Common.Tools.DotNetCore...
Importing namespace Cake.Common.Tools.DotNetCore.Build...
Importing namespace Cake.Common.Tools.DotNetCore.BuildServer...
Importing namespace Cake.Common.Tools.DotNetCore.Clean...
Importing namespace Cake.Common.Tools.DotNetCore.Execute...
Importing namespace Cake.Common.Tools.DotNetCore.MSBuild...
Importing namespace Cake.Common.Tools.DotNetCore.NuGet.Delete...
Importing namespace Cake.Common.Tools.DotNetCore.NuGet.Push...
Importing namespace Cake.Common.Tools.DotNetCore.NuGet.Source...
Importing namespace Cake.Common.Tools.DotNetCore.Pack...
Importing namespace Cake.Common.Tools.DotNetCore.Publish...
Importing namespace Cake.Common.Tools.DotNetCore.Restore...
Importing namespace Cake.Common.Tools.DotNetCore.Run...
Importing namespace Cake.Common.Tools.DotNetCore.Test...
Importing namespace Cake.Common.Tools.DotNetCore.Tool...
Importing namespace Cake.Common.Tools.DotNetCore.VSTest...
Importing namespace Cake.Common.Tools.DupFinder...
Importing namespace Cake.Common.Tools.Fixie...
Importing namespace Cake.Common.Tools.GitLink...
Importing namespace Cake.Common.Tools.GitReleaseManager...
Importing namespace Cake.Common.Tools.GitReleaseManager.AddAssets...
Importing namespace Cake.Common.Tools.GitReleaseManager.Close...
Importing namespace Cake.Common.Tools.GitReleaseManager.Create...
Importing namespace Cake.Common.Tools.GitReleaseManager.Discard...
Importing namespace Cake.Common.Tools.GitReleaseManager.Export...
Importing namespace Cake.Common.Tools.GitReleaseManager.Label...
Importing namespace Cake.Common.Tools.GitReleaseManager.Open...
Importing namespace Cake.Common.Tools.GitReleaseManager.Publish...
Importing namespace Cake.Common.Tools.GitReleaseNotes...
Importing namespace Cake.Common.Tools.GitVersion...
Importing namespace Cake.Common.Tools.ILMerge...
Importing namespace Cake.Common.Tools.ILRepack...
Importing namespace Cake.Common.Tools.InnoSetup...
Importing namespace Cake.Common.Tools.InspectCode...
Importing namespace Cake.Common.Tools.MSBuild...
Importing namespace Cake.Common.Tools.MSpec...
Importing namespace Cake.Common.Tools.MSTest...
Importing namespace Cake.Common.Tools.NSIS...
Importing namespace Cake.Common.Tools.NuGet...
Importing namespace Cake.Common.Tools.NuGet.Add...
Importing namespace Cake.Common.Tools.NuGet.Delete...
Importing namespace Cake.Common.Tools.NuGet.Init...
Importing namespace Cake.Common.Tools.NuGet.Install...
Importing namespace Cake.Common.Tools.NuGet.List...
Importing namespace Cake.Common.Tools.NuGet.Pack...
Importing namespace Cake.Common.Tools.NuGet.Push...
Importing namespace Cake.Common.Tools.NuGet.Restore...
Importing namespace Cake.Common.Tools.NuGet.SetApiKey...
Importing namespace Cake.Common.Tools.NuGet.SetProxy...
Importing namespace Cake.Common.Tools.NuGet.Sources...
Importing namespace Cake.Common.Tools.NuGet.Update...
Importing namespace Cake.Common.Tools.NUnit...
Importing namespace Cake.Common.Tools.OctopusDeploy...
Importing namespace Cake.Common.Tools.OpenCover...
Importing namespace Cake.Common.Tools.ReportGenerator...
Importing namespace Cake.Common.Tools.ReportUnit...
Importing namespace Cake.Common.Tools.Roundhouse...
Importing namespace Cake.Common.Tools.SignTool...
Importing namespace Cake.Common.Tools.SpecFlow...
Importing namespace Cake.Common.Tools.SpecFlow.StepDefinitionReport...
Importing namespace Cake.Common.Tools.SpecFlow.TestExecutionReport...
Importing namespace Cake.Common.Tools.TextTransform...
Importing namespace Cake.Common.Tools.VSTest...
Importing namespace Cake.Common.Tools.VSWhere...
Importing namespace Cake.Common.Tools.VSWhere.All...
Importing namespace Cake.Common.Tools.VSWhere.Latest...
Importing namespace Cake.Common.Tools.VSWhere.Legacy...
Importing namespace Cake.Common.Tools.VSWhere.Product...
Importing namespace Cake.Common.Tools.WiX...
Importing namespace Cake.Common.Tools.WiX.Heat...
Importing namespace Cake.Common.Tools.XBuild...
Importing namespace Cake.Common.Tools.XUnit...
Importing namespace Cake.Common.Xml...
Importing namespace Cake.Core...
Importing namespace Cake.Core.Diagnostics...
Importing namespace Cake.Core.IO...
Importing namespace Cake.Core.Scripting...
Importing namespace Cake.Docker...
Importing namespace Cake.Yarn...
Importing namespace System...
Importing namespace System.Collections.Generic...
Importing namespace System.IO...
Importing namespace System.Linq...
Importing namespace System.Text...
Importing namespace System.Threading.Tasks...
Compiling build script...
C:/git/.../build.cake(34,56): error CS1061: 'GitVersion' does not contain a definition for 'EscapedBranchName' and no accessible extension method 'EscapedBranchName' accepting a first argument of type 'GitVersion' could be found (are you missing a using directive or an assembly reference?)
Error: Cake.Core.CakeException: Error(s) occurred when compiling build script:
C:/git/.../build.cake(34,56): error CS1061: 'GitVersion' does not contain a definition for 'EscapedBranchName' and no accessible extension method 'EscapedBranchName' accepting a first argument of type 'GitVersion' could be found (are you missing a using directive or an assembly reference?)
   at Cake.Scripting.Roslyn.RoslynScriptSession.Execute(Script script)
   at Cake.Core.Scripting.ScriptRunner.Run(IScriptHost host, FilePath scriptPath, IDictionary`2 arguments)
   at Cake.Commands.BuildCommand.Execute(CakeOptions options)
   at Cake.CakeApplication.Run(CakeOptions options)
   at Cake.Program.Main()
```

--- a/src/Cake.Common.Tests/Unit/Tools/GitVersion/GitVersionRunnerTests.cs
+++ b/src/Cake.Common.Tests/Unit/Tools/GitVersion/GitVersionRunnerTests.cs
@@ -270,7 +270,9 @@ namespace Cake.Common.Tests.Unit.Tools.GitVersion
                         PreReleaseTag = "PreReleaseTag",
                         PreReleaseTagWithDash = "PreReleaseTagWithDash",
                         PreReleaseLabel = "PreReleaseLabel",
+                        PreReleaseLabelWithDash = "-PreReleaseLabel",
                         PreReleaseNumber = null,
+                        WeightedPreReleaseNumber = null,
                         BuildMetaData = "BuildMetaData",
                         BuildMetaDataPadded = "BuildMetaDataPadded",
                         FullBuildMetaData = "Branch.master.Sha.f2467748c78b3c8b37972ad0b30df2e15dfbf2cb",
@@ -283,12 +285,17 @@ namespace Cake.Common.Tests.Unit.Tools.GitVersion
                         FullSemVer = "0.1.1",
                         InformationalVersion = "0.1.1+Branch.master.Sha.f2467748c78b3c8b37972ad0b30df2e15dfbf2cb",
                         BranchName = "master",
+                        EscapedBranchName = "master",
                         Sha = "f2467748c78b3c8b37972ad0b30df2e15dfbf2cb",
                         ShortSha = "f2467748",
                         NuGetVersionV2 = "0.1.1",
                         NuGetVersion = "0.1.1",
+                        NuGetPreReleaseTagV2 = "tag",
+                        NuGetPreReleaseTag = "tag",
+                        VersionSourceSha = "f2467748c78b3c8b37972ad0b30df2e15dfbf2cb",
                         CommitsSinceVersionSource = null,
                         CommitsSinceVersionSourcePadded = "0002",
+                        UncommittedChanges = 0,
                         CommitDate = "2017-09-13",
                     }
                     ;
@@ -302,7 +309,9 @@ namespace Cake.Common.Tests.Unit.Tools.GitVersion
                         "  \"PreReleaseTag\":\"PreReleaseTag\",",
                         "  \"PreReleaseTagWithDash\":\"PreReleaseTagWithDash\",",
                         "  \"PreReleaseLabel\":\"PreReleaseLabel\",",
+                        "  \"PreReleaseLabelWithDash\":\"-PreReleaseLabel\",",
                         "  \"PreReleaseNumber\":\"\",",
+                        "  \"WeightedPreReleaseNumber\":\"\",",
                         "  \"BuildMetaData\":\"BuildMetaData\",",
                         "  \"BuildMetaDataPadded\":\"BuildMetaDataPadded\",",
                         "  \"FullBuildMetaData\":\"Branch.master.Sha.f2467748c78b3c8b37972ad0b30df2e15dfbf2cb\",",
@@ -315,12 +324,17 @@ namespace Cake.Common.Tests.Unit.Tools.GitVersion
                         "  \"FullSemVer\":\"0.1.1\",",
                         "  \"InformationalVersion\":\"0.1.1+Branch.master.Sha.f2467748c78b3c8b37972ad0b30df2e15dfbf2cb\",",
                         "  \"BranchName\":\"master\",",
+                        "  \"EscapedBranchName\":\"master\",",
                         "  \"Sha\":\"f2467748c78b3c8b37972ad0b30df2e15dfbf2cb\",",
                         "  \"ShortSha\":\"f2467748\",",
                         "  \"NuGetVersionV2\":\"0.1.1\",",
                         "  \"NuGetVersion\":\"0.1.1\",",
+                        "  \"NuGetPreReleaseTagV2\":\"tag\",",
+                        "  \"NuGetPreReleaseTag\":\"tag\",",
+                        "  \"VersionSourceSha\":\"f2467748c78b3c8b37972ad0b30df2e15dfbf2cb\",",
                         "  \"CommitsSinceVersionSource\":\"\",",
                         "  \"CommitsSinceVersionSourcePadded\":\"0002\",",
+                        "  \"UncommittedChanges\":\"0\",",
                         "  \"CommitDate\":\"2017-09-13\"",
                         "}"
                     });
@@ -336,7 +350,9 @@ namespace Cake.Common.Tests.Unit.Tools.GitVersion
                 Assert.Equal(expect.PreReleaseTag, result.PreReleaseTag);
                 Assert.Equal(expect.PreReleaseTagWithDash, result.PreReleaseTagWithDash);
                 Assert.Equal(expect.PreReleaseLabel, result.PreReleaseLabel);
+                Assert.Equal(expect.PreReleaseLabelWithDash, result.PreReleaseLabelWithDash);
                 Assert.Equal(expect.PreReleaseNumber, result.PreReleaseNumber);
+                Assert.Equal(expect.WeightedPreReleaseNumber, result.WeightedPreReleaseNumber);
                 Assert.Equal(expect.BuildMetaData, result.BuildMetaData);
                 Assert.Equal(expect.BuildMetaDataPadded, result.BuildMetaDataPadded);
                 Assert.Equal(expect.FullBuildMetaData, result.FullBuildMetaData);
@@ -349,12 +365,17 @@ namespace Cake.Common.Tests.Unit.Tools.GitVersion
                 Assert.Equal(expect.FullSemVer, result.FullSemVer);
                 Assert.Equal(expect.InformationalVersion, result.InformationalVersion);
                 Assert.Equal(expect.BranchName, result.BranchName);
+                Assert.Equal(expect.EscapedBranchName, result.EscapedBranchName);
                 Assert.Equal(expect.Sha, result.Sha);
                 Assert.Equal(expect.ShortSha, result.ShortSha);
                 Assert.Equal(expect.NuGetVersionV2, result.NuGetVersionV2);
                 Assert.Equal(expect.NuGetVersion, result.NuGetVersion);
+                Assert.Equal(expect.NuGetPreReleaseTagV2, result.NuGetPreReleaseTagV2);
+                Assert.Equal(expect.NuGetPreReleaseTag, result.NuGetPreReleaseTag);
+                Assert.Equal(expect.VersionSourceSha, result.VersionSourceSha);
                 Assert.Equal(expect.CommitsSinceVersionSource, result.CommitsSinceVersionSource);
                 Assert.Equal(expect.CommitsSinceVersionSourcePadded, result.CommitsSinceVersionSourcePadded);
+                Assert.Equal(expect.UncommittedChanges, result.UncommittedChanges);
                 Assert.Equal(expect.CommitDate, result.CommitDate);
             }
 

--- a/src/Cake.Common/Tools/GitVersion/GitVersion.cs
+++ b/src/Cake.Common/Tools/GitVersion/GitVersion.cs
@@ -40,9 +40,19 @@ namespace Cake.Common.Tools.GitVersion
         public string PreReleaseLabel { get; set; }
 
         /// <summary>
+        /// Gets or sets the pre-release label with dash.
+        /// </summary>
+        public string PreReleaseLabelWithDash { get; set; }
+
+        /// <summary>
         /// Gets or sets the pre-release number.
         /// </summary>
         public int? PreReleaseNumber { get; set; }
+
+        /// <summary>
+        /// Gets or sets the weighted pre-release number.
+        /// </summary>
+        public int? WeightedPreReleaseNumber { get; set; }
 
         /// <summary>
         /// Gets or sets the build metadata.
@@ -105,6 +115,11 @@ namespace Cake.Common.Tools.GitVersion
         public string BranchName { get; set; }
 
         /// <summary>
+        /// Gets or sets the escaped branch name.
+        /// </summary>
+        public string EscapedBranchName { get; set; }
+
+        /// <summary>
         /// Gets or sets the Git SHA.
         /// </summary>
         public string Sha { get; set; }
@@ -125,6 +140,21 @@ namespace Cake.Common.Tools.GitVersion
         public string NuGetVersion { get; set; }
 
         /// <summary>
+        /// Gets or sets the NuGet pre-release tag for v2.
+        /// </summary>
+        public string NuGetPreReleaseTagV2 { get; set; }
+
+        /// <summary>
+        /// Gets or sets the NuGet pre-release tag.
+        /// </summary>
+        public string NuGetPreReleaseTag { get; set; }
+
+        /// <summary>
+        /// Gets or sets the version source sha.
+        /// </summary>
+        public string VersionSourceSha { get; set; }
+
+        /// <summary>
         /// Gets or sets the commits since version source.
         /// </summary>
         public int? CommitsSinceVersionSource { get; set; }
@@ -133,6 +163,11 @@ namespace Cake.Common.Tools.GitVersion
         /// Gets or sets the commits since version source padded.
         /// </summary>
         public string CommitsSinceVersionSourcePadded { get; set; }
+
+        /// <summary>
+        /// Gets or sets the number of uncommited changes.
+        /// </summary>
+        public int? UncommittedChanges { get; set; }
 
         /// <summary>
         /// Gets or sets the commit date.

--- a/src/Cake.Common/Tools/GitVersion/GitVersionInternal.cs
+++ b/src/Cake.Common/Tools/GitVersion/GitVersionInternal.cs
@@ -57,10 +57,24 @@ namespace Cake.Common.Tools.GitVersion
         }
 
         [DataMember]
+        public string PreReleaseLabelWithDash
+        {
+            get => GitVersion.PreReleaseLabelWithDash;
+            set => GitVersion.PreReleaseLabelWithDash = value;
+        }
+
+        [DataMember]
         public string PreReleaseNumber
         {
             get => ToString(GitVersion.PreReleaseNumber);
             set => GitVersion.PreReleaseNumber = ToNullableInt(value);
+        }
+
+        [DataMember]
+        public string WeightedPreReleaseNumber
+        {
+            get => ToString(GitVersion.WeightedPreReleaseNumber);
+            set => GitVersion.WeightedPreReleaseNumber = ToNullableInt(value);
         }
 
         [DataMember]
@@ -148,6 +162,13 @@ namespace Cake.Common.Tools.GitVersion
         }
 
         [DataMember]
+        public string EscapedBranchName
+        {
+            get => GitVersion.EscapedBranchName;
+            set => GitVersion.EscapedBranchName = value;
+        }
+
+        [DataMember]
         public string Sha
         {
             get => GitVersion.Sha;
@@ -176,6 +197,27 @@ namespace Cake.Common.Tools.GitVersion
         }
 
         [DataMember]
+        public string NuGetPreReleaseTagV2
+        {
+            get => GitVersion.NuGetPreReleaseTagV2;
+            set => GitVersion.NuGetPreReleaseTagV2 = value;
+        }
+
+        [DataMember]
+        public string NuGetPreReleaseTag
+        {
+            get => GitVersion.NuGetPreReleaseTag;
+            set => GitVersion.NuGetPreReleaseTag = value;
+        }
+
+        [DataMember]
+        public string VersionSourceSha
+        {
+            get => GitVersion.VersionSourceSha;
+            set => GitVersion.VersionSourceSha = value;
+        }
+
+        [DataMember]
         public string CommitsSinceVersionSource
         {
             get => ToString(GitVersion.CommitsSinceVersionSource);
@@ -187,6 +229,13 @@ namespace Cake.Common.Tools.GitVersion
         {
             get => GitVersion.CommitsSinceVersionSourcePadded;
             set => GitVersion.CommitsSinceVersionSourcePadded = value;
+        }
+
+        [DataMember]
+        public string UncommittedChanges
+        {
+            get => ToString(GitVersion.UncommittedChanges);
+            set => GitVersion.UncommittedChanges = ToNullableInt(value);
         }
 
         [DataMember]

